components/signup/signup.scss

```diff
@@ -0,0 +1,387 @@
+                    }
+
+                    .signup-body-card-form-button-submit {
+                        @include primary-button;
+                        @include button-large;
+
+                        width: 100%;
+                        margin-top: 30px;
+                    }
+                }
+
+                .signup-body-card-form-divider {
+                    display: flex;
+                    height: 1px;
+                    justify-content: center;
+                    margin: 40px 0;
+                    background: rgba(var(--center-channel-text-rgb), 0.08);
+                    text-align: center;
+
+                    .signup-body-card-form-divider-label {
+                        padding: 0 10px;
+                        background-color: var(--center-channel-bg);
+                        color: rgba(var(--center-channel-text-rgb), 0.56);
+                        font-size: 12px;
+                        line-height: 0;
+                    }
+                }
+
+                .signup-body-card-form-login-options {
+                    display: flex;
+                    flex: 1;
+                    flex-wrap: wrap;
+                    justify-content: space-between;
+                    column-gap: 8px;
+                    row-gap: 8px;
+
+                    &.column {
+                        flex-direction: column;
+                        margin-top: 22px;
+                        row-gap: 24px;
+                    }
+                }
+
+                .signup-body-card-agreement {
+                    margin-top: 32px;
+                    color: rgba(var(--center-channel-text-rgb), 0.56);
+                    font-size: 11px;
+                    line-height: 16px;
+
+                    a {
+                        @include link;
+
+                        font-size: 11px;
+                    }
+                }
+            }
+        }
+
+        .signup-body-custom-branding-markdown {
+            flex: 1;
+            margin-top: 28px;
+
+            ul + p,
+            ol + p {
+                margin-top: 0.6em;
+            }
+
+            p + ul,
+            p + ol {
+                margin-top: 0.6em;
+            }
+
+            img {
+                max-width: 450px;
+            }
+
+            p {
+                width: 100%;
+                margin: 0;
+                color: rgba(var(--center-channel-text-rgb), 0.72);
+                font-size: 18px;
+                font-weight: 400;
+                line-height: 28px;
+                white-space: pre-wrap;
+                word-break: break-word;
+            }
+        }
+
+        .signup-body-content-button-container {
+            display: flex;
+            flex: 1;
+            justify-content: center;
+
+            .signup-body-content-button-return {
+                @include primary-button;
+                @include button-medium;
+            }
+        }
+    }
+}
+
+@keyframes shake {
```

Comment:
nit: would be nice if this is in a mixin

Reply:
Updated